schemas/schemas.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package schemas
+
+import (
+	"embed"
+)
+
+// FS contains the known schemas for OPA's Authorization Policy etc.
+// "authorizationPolicy.json" contains the input schema for OPA's Authorization Policy
+//
+//go:embed *.json
+var FS embed.FS
```

Comment:
> Since we're always populating compiler.schemaDefinitions on init, the file will always stick around in memory, though, won't it?

I think so. It's a small file (~4KB) so I don't imagine it to have a significant memory impact on OPA. Is that the concern here? If it does we can look into conditionally loading it but does not seem the case atm.

Reply:
No, I don't think memory consumption is much of a concern here.